### ISSUE TITLE
Component blueprint generates deprecation warning

### ISSUE THREAD

**Issue by downie** [san650/ember-cli-page-object#77]
Making a new page object component with this command:
```
ember generate page-object-component my-component
```
generates code that uses `PageObject.component` which gives the following deprecation message:
```
`component` is deprecated in favor of using plain JavaScript objects
```

What should the blueprint generate for components to avoid that deprecation warning? something like this?
```
export default {
  title: text('h3')
};
```

**Comment by san650** [comment]
@downie yes, exactly that!

Can you create a pull request with that solution?

**Comment by downie** [comment]
will do

**Comment by san650** [comment]
Fixed by #78 

Thanks for reporting and fixing the error
